src/sentry/release_health/tasks.py

```diff
@@ -28,12 +29,12 @@
 
 
 @instrumented_task(
-    name="sentry.tasks.monitor_release_adoption",
+    name="sentry.release_health.tasks.monitor_release_adoption",
     queue="releasemonitor",
     default_retry_delay=5,
     max_retries=5,
-)
-def monitor_release_adoption(**kwargs):
+)  # type: ignore
+def monitor_release_adoption(**kwargs) -> None:
```

Comment:
`-> None`, are we starting to add these for functions that return nothing?

Reply:
Mypy was complaining at me about this so I just added the type annotation